stage.js

```diff
@@ -29,7 +30,19 @@
 
     checkForLink(e.target);
   };
-  
+
+  var zoom = function () { win.zoomLevel += 1; }
```

Comment:
Putting these functions inside `handleKeydown`, like done in `handleClick`, would redefine them every time a key is pressed.